src/utils.py

```diff
@@ -206,14 +206,16 @@ async def get_repo(name: str) -> github3.repos.repo.Repository:
     return gh.repository("PrefectHQ", name)
 
 
-def validate_view_content(view_dict: dict, variety: CollectionViewName) -> None:
+def validate_view_content(view_dict: dict, variety: CollectionViewVariety) -> None:
     """Raises an error if the view content is not valid."""
     schema = getattr(metadata_schemas, f"{variety}_schema")
     validate = fastjsonschema.compile(schema)
 
     for collection_name, collection_metadata in view_dict.items():
+        if variety == "block":
+            collection_metadata = collection_metadata["block_types"]
         try:
-            validate(list(collection_metadata.values())[0])
+            list(map(validate, collection_metadata.values()))
```

Comment:
I don't think the list is necessary now? 
```suggestion
            map(validate, collection_metadata.values())
```

Reply:
wait, I just saw the commit message for [97e1818](https://github.com/PrefectHQ/prefect-collection-registry/pull/174/commits/97e1818b09dd0727d7dca24b322fdc4458cd244b). Maybe there's a different function that doesn't need to be converted to a list?

Reply:
Forget what I said. This is good.